scalanativelib/test/src/HelloNativeWorldTests.scala

```diff
@@ -22,12 +23,12 @@ object HelloNativeWorldTests extends TestSuite {
     override def mainClass = Some("hello.Main")
   }
 
-  val scala213 = "2.13.6"
+  val scala213 = "2.13.7"
```

Comment:
We probably might want to derive this from what is defined in `build.sc` (for easier maintenance), either by generated build info classes or by system properties. An example can be found in `contrib.scoverage`.

Reply:
I noticed that I can't update this version since Scala Native 0.4.0 is not published for newer Scala versions.
I reverted the change.